.github/workflows/update-contributors.yaml

```diff
@@ -3,7 +3,7 @@ name: Update Contributors
 on:
   push:
     branches:
-      - main
+      - doesnotexist
```

Comment:
What is the purpose of the branch change?

Reply:
Right now the all contributors workflow checks for new contributors and updates the metadata file. However because the commit isn't signed, it will never pass CI (see https://github.com/oras-project/oras-py/pull/18) so I'm disabling it to run for now (via a branch that doesn't exist) so we can discuss how to either fix it, allow an exception, or remove it.